pages/pricing/_support.html

```diff
@@ -0,0 +1,99 @@
+<div id="support" class="row row-page-header">
```

Comment:
Include an `<hr>` right above this. It's currently missing on the page.

Reply:
Good catch. Fixed: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/23/commits/87d038d4e7c0b74c14e4a035bb2cddc781af3779